src/shared/filesystemUtilities.ts

```diff
@@ -220,3 +221,21 @@ export async function cloud9Findfile(dir: string, fileName: string): Promise<vsc
     }
     return []
 }
+/**
+ * @returns  A string path to the last locally stored download location. If none, returns the users 'Downloads' directory path.
+ */
+export function getDefaultDownloadPath(): string {
+    const lastUsedPath = globals.context.globalState.get('aws.downloadPath')
+    if (lastUsedPath) {
+        if (typeof lastUsedPath === 'string') {
+            return lastUsedPath
+        } else {
+            getLogger().verbose("Expected stored 'aws.downloadPath' to return a string")
+        }
```

Comment:
Don't need else when the other branch exits the scope (throw, return, etc.). Please use this pattern for all future code.

```suggestion
        getLogger().verbose("Expected stored 'aws.downloadPath' to return a string")
```